### DISCUSSION TITLE
Plugin that could show inventory grid lines or click boxes for each inv slot (without having to drag items around)

### DISCUSSION THREAD

**Discussion by Dell63** [runelite/runelite#18193] [Ideas]
I was trying to look for some 3rd party soft wear that could do this, but I haven't had much luck finding anything yet that would work well for RL -although I'm sure it's out there somewhere, it would be really nice if there was a plugin that could just show the click boxes for my inventory just like I have in the screenshot (this only appears due to the 'inventory grid' plugin as I'm trying to drag items around)
edit: (i've tried using screen markers before but they show through to other tabs + don't stay in 1 spot consistently as I drag the screen around)
![image](https://github.com/user-attachments/assets/9a887ae9-332b-4da8-847d-6a8e7e402234)

**Comment by TV-cmd** [comment]
very good
